app/controllers/menu/category_controller.rb

```diff
@@ -0,0 +1,10 @@
+class Menu::CategoryController < ApplicationController
+  def show
+    @items = current_category.items
+  end
+
+  def current_category
+    Category.find_by(name: params[:id])
```

Comment:
I'm not sure what comes in with `params`, but shouldn't this be `(name: params[:name])` instead of `:id`?